packages/sandbox/src/app/sandbox/docs/shadowsValidation.doc.tsx

```diff
@@ -0,0 +1,136 @@
+            of the mind's eye invent the universe not a sunrise but a
+            galaxyrise. The ash of stellar alchemy another world not a sunrise
+            but a galaxyrise corpus callosum the carbon in our apple pies made
+            in the interiors of collapsing stars. Worldlets citizens of distant
+            epochs emerged into consciousness paroxysm of global death preserve
+            and cherish that pale blue dot star stuff harvesting star light.
+          </Text>
+          <Box paddingVertical="sm">
+            <Card backgroundColor="BLUKO_500" shadows="strong">
+              <Text color="PURE_WHITE_1000">Strong</Text>
+            </Card>
+          </Box>
+          <Text>
+            Paroxysm of global death permanence of the stars are creatures of
+            the cosmos vastness is bearable only through love dream of the
+            mind's eye paroxysm of global death? Consciousness citizens of
+            distant epochs a billion trillion vastness is bearable only through
+            love Tunguska event Jean-François Champollion. A still more glorious
+            dawn awaits a mote of dust suspended in a sunbeam something
+            incredible is waiting to be known vanquish the impossible
+            intelligent beings laws of physics. Dispassionate extraterrestrial
+            observer brain is the seed of intelligence star stuff harvesting
+            star light network of wormholes tendrils of gossamer clouds courage
+            of our questions. Inconspicuous motes of rock and gas as a patch of
+            light Vangelis tendrils of gossamer clouds take root and flourish
+            Cambrian explosion? Two ghostly white figures in coveralls and
+            helmets are softly dancing paroxysm of global death courage of our
+            questions a mote of dust suspended in a sunbeam hearts of the stars
+            citizens of distant epochs. Two ghostly white figures in coveralls
+            and helmets are softly dancing paroxysm of global death Hypatia
+            concept of the number one Sea of Tranquility intelligent beings and
+            billions upon billions upon billions upon billions upon billions
+            upon billions upon billions.
+          </Text>
+        </Box>
+      </ScrollView>
+      <Box
+        position="absolute"
+        bottom={30}
+        left={0}
+        right={0}
+        pointerEvents="none"
+      >
+        <ButtonDock
+          isFloating
+          primary={{
+            text: 'Danger Default',
+            onPress: () => {
+              console.log('Primary button pressed');
+            },
+            appearance: 'danger',
+          }}
+          secondary={{
+            text: 'Neutral Default',
+            onPress: () => {
+              console.log('Primary button pressed');
+            },
+            appearance: 'neutral',
+          }}
+        />
+      </Box>
+    </Box>
+  );
+};
```

Comment:
@ianrobertlambert did you have time to review shadows on your devices so I can delete this examples?
![Screenshot 2023-04-25 at 18 40 32](https://user-images.githubusercontent.com/1573707/235174058-fbd295c1-4415-42e2-a6cf-25fed8935d3b.png)

https://qr.expo.dev/eas-update?projectId=11eb7a11-ac37-40c7-a649-095da5a7b0b3&groupId=b5e8e929-d2aa-46e7-81a5-54aeaa8322b2